scripts/schemas/patch-schema.yml

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# A pykwalify schema for basic validation of the patches.yml format.
+
+# The schema for individual patch objects
+schema;patch-schema:
+    type: seq
+    sequence:
+      - type: map
+        mapping:
+          # The path to the patch file, relative to the root of the module
+          # E.g. zephyr/kernel-pipe-fix-not-k-no-wait-and-ge-min-xfer-bytes.patch
+          path:
+            required: true
+            type: str
+          # The SHA-256 checksum of the patch file
+          # e.g. e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+          sha256sum:
+            required: true
+            type: str
+            pattern: "^[0-9a-f]{64}$"
+          # The name of the module the patch is for e.g. zephyr, or bootloader/mcuboot
```

Comment:
Is it the name or path? The comment suggests the latter.

Reply:
path

Reply:
```suggestion
          # The path of the module the patch is for e.g. zephyr, or bootloader/mcuboot
```

Reply:
Fixed!